src/server.js

```diff
@@ -8,9 +8,11 @@ const { sequelize } = require('./models');
 const { artistRoutes } = require('./routes/artist-routes');
 const { showRoutes } = require('./routes/show-routes');
 const { authRoutes, checkJWT } = require('../auth');
+const { blogRoutes } = require('./routes/blog-routes');
 
 app.use(express.json());
 app.use(authRoutes);
+app.use(blogRoutes);
 app.use(artistRoutes);
 app.use(showRoutes);
```

Comment:
Move these before authRoutes, and then move the 
````
app.use(artistRoutes);
app.use(showRoutes);
app.use(authRoutes);
app.use(blogRoutes);
```` 
To below the `/loggedin` route. Hopefully, that will get your server test to pass again